### PR TITLE
Changed testing image for insights-operator

### PR DESCRIPTION
**Pull request by quarckster**
Insights operator tests are packed in a separate container image. This will allow us to write tests on a different programming language and use dependencies that are not publicly available.

DO NOT MERGE.

### CODE REVIEW

**Comment by quarckster** [comment]
retest

**Comment by quarckster** [comment]
@tremes please review. This PR changes tests for insights operator.

File: ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml [hunk 1]
```diff
@@ -72,29 +72,13 @@ tests:
     test:
     - as: test
       cli: latest
-      commands: make test-e2e
-      from: src
+      commands: pytest.sh -m "not ui"
+      from: insights-operator-tests
```
Comment (tremes) [inline ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml:76]: @quarckster just out of curiosity. What is building and pushing the `quay.io/redhatqe/insights-operator-tests` image pls (I vaguely remember asking the question so sorry :) )? Gitlab CI?

Also the `pytest.sh -m "not ui"` - is there some UI option? :)

File: ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml [hunk 2]
```diff
@@ -72,29 +72,13 @@ tests:
...
-    test:
-    - as: test
-      cli: latest
-      commands: make test-e2e
-      from: src
-      resources:
-        requests:
-          cpu: 100m
-    workflow: ipi-gcp
-- as: insights-operator-tests-periodic
```
Comment (tremes) [inline ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml:97]: OK so this is removing the original integration test execution step right? I guess we can remove the integration tests from the IO repo once this is merged. WDYT?

File: ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml (hunk 1 above)
Comment (quarckster) [reply ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml:76]: The image is built here https://gitlab.cee.redhat.com/ccx/insights-operator-tests/-/blob/master/.gitlab-ci.yml. And, yes, these new tests can run UI tests against OCP web console. There three of them in the test suite.

File: ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml (hunk 2 above)
Comment (quarckster) [reply ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml:97]: Right, this removes nightly test runs that use tests from IO repo. And yes, this directory can be removed https://github.com/openshift/insights-operator/tree/master/tests.

**Comment by tremes** [comment]
I quickly went through the new test repo and it looked good to me. Thanks a lot.
/lgtm

**Pull request merged**